From e29badd2c57b689827951cf7e21337810023cc2d Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Wed, 23 Sep 2015 10:24:51 +0800
Subject: [PATCH 0499/5242] MLK-11612-02 ARM: dts: imx6sx-sabreauto: add spidf
 sound card support

commit  896d651a1e5ffc4faf88a4af3a3e3a0293ab2fef from
https://source.codeaurora.org/external/imx/linux-imx.git

add spidf sound card support for imx6sx-sabreauto board

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |   19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 0c57386..db2e492 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -90,6 +90,13 @@
 		asrc-controller = <&asrc>;
 		audio-codec = <&codec>;
 	};
+
+	sound-spdif {
+		compatible = "fsl,imx-audio-spdif";
+		model = "imx-spdif";
+		spdif-controller = <&spdif>;
+		spdif-in;
+	};
 };
 
 &clks {
@@ -235,6 +242,12 @@
 	status = "okay";
 };
 
+&spdif {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_spdif_3>;
+	status = "okay";
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1>;
@@ -426,6 +439,12 @@
 		>;
 	};
 
+	pinctrl_spdif_3: spdifgrp-3 {
+		fsl,pins = <
+			MX6SX_PAD_ENET2_COL__SPDIF_IN           0x1b0b0
+		>;
+	};
+
 	pinctrl_uart1: uart1grp {
 		fsl,pins = <
 			MX6SX_PAD_GPIO1_IO04__UART1_TX		0x1b0b1
-- 
1.7.9.5

